includes/api/class-edd-api.php

```diff
@@ -101,6 +137,15 @@ class EDD_API {
 	 * @since 1.5
 	 */
 	public function __construct() {
+
+		$this->versions = array(
+			'v1' => 'EDD_API_V1',
+		);
+
+		foreach( $this->versions as $version => $class ) {
+			require_once EDD_PLUGIN_DIR . 'includes/api/class-edd-api-' . $version . '.php';
```

Comment:
Considered that but it's faster to not do it, and this is no different that our main EDD includes. This files are not filterable so the only reason one would disappear is if it is deleted, which would result in bad things even with the check.